static/coffee/controllers.coffee

```diff
@@ -843,6 +845,6 @@ controllers.controller('PodController', ['$scope', 'PodApi', ($scope, PodApi) ->
     # TODO show failure message
 
   $scope.onTriggerSuccess = () ->
-    # TODO update notes
+    $scope.$emit('podActionSuccess')
```

Comment:
On second though, I kind of agree, `CaseController` shouldn't need to know that a successful pod action trigger means the timeline has changed. I'm going to rather `$emit` a `timelineChanged` event from the pod (since it 'knows' the timeline has changed), and have `CaseController` listen and `$broadcast` it.